packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -451,6 +468,12 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
         commitAttachRef(nextEffect);
       }
 
+      if (enableProfileModeMetrics) {
+        if (effectTag & CommitProfile) {
```

Comment:
Can you reuse the `Update` effect type? I don't think we need a new tag here.

Reply:
Profiler is the only type of work that might have this effect, right?

Reply:
Hm. Maybe. Not obvious to me why that would be an improvement.

Reply:
Our comments got crossed.

> Profiler is the only type of work that might have this effect, right?

Yes.

Reply:
You're adding an extra check to every single unit of work. We already check for the `Update` type.

Reply:
👍